crates/re_viewer/src/ui/welcome_screen/welcome_page.rs

```diff
@@ -231,3 +260,38 @@ fn image_banner(ui: &mut egui::Ui, icon: &re_ui::Icon, column_width: f32, max_im
         );
     });
 }
+
+fn open_quick_start<'a>(
+    command_sender: &re_viewer_context::CommandSender,
+    parts: impl IntoIterator<Item = &'a str>,
+    app_id: impl AsRef<str>,
+    entity_path: impl AsRef<str>,
+) {
+    let markdown = parts.into_iter().join("\n");
+    let res = open_markdown_recording(command_sender, markdown, app_id, entity_path);
+    if let Err(err) = res {
+        re_log::error!("Failed to load quick start: {}", err);
+    }
+}
+
+fn open_markdown_recording(
+    command_sender: &re_viewer_context::CommandSender,
+    markdown: impl AsRef<str>,
+    app_id: impl AsRef<str>,
+    entity_path: impl AsRef<str>,
+) -> anyhow::Result<()> {
+    let text_doc = re_types::archetypes::TextDocument::new(markdown.as_ref())
```

Comment:
nit/style: we don't need to make private functions like these unnecessarily complex

```suggestion
    markdown: &str,
    app_id: &str,
    entity_path: &str,
) -> anyhow::Result<()> {
    let text_doc = re_types::archetypes::TextDocument::new(markdown)
```